server/shared/types/routes/termdb.getnumericcategories.ts

```diff
@@ -0,0 +1,22 @@
+import { Filter } from '../filter'
+
+export type getnumericcategoriesRequest = {
+	/** a user-defined genome label in the serverconfig.json, hg38, hg19, mm10, etc */
+	genome: string
+	/** a user-defined dataset label in the serverconfig.json, such as ClinVar, SJLife, GDC, etc */
+	dslabel: string
+	embedder: string
+	getnumericcategories: number
```

Comment:
this property is no longer needed, please delete here and also from vocab `getNumericUncomputableCategories()` arg